_includes/nav.html

```diff
@@ -1,41 +1,160 @@
-<header class="page">
-    {% if page.fixedHeader != true %}
-    <div class="grid">
-        <div class="grid-1-1 page-title">
+<header class="page{% if page.fixedHeader %} headFixed{% endif %} {% if page.image %}hasImage{% endif %}" {% if page.image %}style="background-image: url('/images/{{ page.image }}');"{% endif %}>
+
+    <nav class="sticky">
+
+        <div class="page-title">
             <a href="/">
-                <svg class="sbg-logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-                    <use xlink:href="/images/logos/skybettingandgaming.svg#logo"></use>
-                </svg>
+            <svg class="sbg-logo" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"

```

Comment:
That's a shame: it means that instead of the logo being loaded once and being cached, it's loaded every page load.  Won't stop us launching, but it would definitely be worth looking at trying to fix that if we can.